models/gmx/arbitrum/vault/gmx_arbitrum_vault_balances.sql

```diff
@@ -38,9 +36,7 @@ vault_balances_frax AS -- This CTE returns the balance of FRAX tokens in the GMX
             FROM {{ source('erc20_arbitrum', 'evt_transfer') }}
             WHERE `to` = '0x489ee077994b6658eafa855c308275ead8097c4a' -- GMX Arbitrum Vault Address
             AND `contract_address` = '0x17fc002b466eec40dae837fc4be5c67993ddbd6f' -- FRAX Arbitrum Smart Contract
-            {% if is_incremental() %}
             AND evt_block_time >= date_trunc("day", now() - interval '1 week')
```

Comment:
if this is a view, i don't believe you'll want these filters for 1 week, otherwise you'll have a bunch of nulls for history again since minute goes back to 2021 but these all go back one week